Jenkinsfile

```diff
@@ -167,7 +167,7 @@ node('docker && imgtec') {  // Only run on internal slaves as build takes a lot
                 try {
                     sh "cp ${env.PRIVATE_KEY} ${WORKSPACE}/key-build"
                     sh "cp ${env.PUBLIC_KEY} ${WORKSPACE}/key-build.pub"
-                    sh "make -j4 V=s ${params.ALL_PACKAGES ? 'IGNORE_ERRORS=m' : ''}"
+                    sh "make -j1 V=s ${params.ALL_PACKAGES ? 'IGNORE_ERRORS=m' : ''}"
```

Comment:
For testing purposes for now, we've had issues with the all packages build buildings packages randomly, i.e. libunwind would build in some cases and fail in other. it seems depending when the -j4 build fails it will drop some packages.